src/open_producten/locaties/migrations/0005_alter_locatie_postcode_alter_organisatie_postcode.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.17 on 2025-03-03 10:25
+
+from django.db import migrations, models
+import open_producten.utils.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("locaties", "0004_alter_locatie_huisnummer_alter_locatie_naam_and_more"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="locatie",
+            name="postcode",
+            field=models.CharField(
+                blank=True,
+                max_length=7,
+                null=True,
+                validators=[
+                    open_producten.utils.validators.CustomRegexValidator(
+                        message="Invalid postal code (spaces are not allowed).",
+                        regex="^[1-9][0-9]{3}[a-zA-Z]{2}$",
+                    )
```

Comment:
Recently in OpenKlant I added the same validator, but only allowing uppercase values and only with a space
```suggestion
                        message="Invalid postal code (spaces are not allowed).",
                        regex="^[1-9][0-9]{3} [A-Z]{2}$",
                    )
```